src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -79,4 +80,10 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+  /** Returns the total number of messages for all users. */
+  public int getTotalMessageCount(){
+    Query query = new Query("Message");
```

Comment:
What does this "Message" stand for? Should it be "Messages"?

Reply:
If I'm not mistaken, "Message" is what gets searched for, specifically by entity, so if we wanted this to work, I think we would have to change the Entity name earlier (around line 42) to "Messages" as well? I was thinking it may make more sense to leave it as is, because each entity is an individual message, but there is also a chance I am totally misinterpreting this. If so, just let me know!

Reply:
SGTM! Thanks for the explanation!